pkg/csi_driver/utils_test.go

```diff
@@ -129,6 +129,11 @@ func TestIsSidecarVersionSupportedForTokenServer(t *testing.T) {
 				imageName:         "us-central1-artifactregistry.gcr.io/gke-release/gke-release/gcs-fuse-csi-driver-sidecar-mounter:v1.18.3-gke.2@sha256:abcd",
 				expectedSupported: true,
 			},
+			{
+				name:              "should return true for supported sidecar version with container registory gke.gcr.io",
+				imageName:         "gke.gcr.io/gcs-fuse-csi-driver-sidecar-mounter:v1.18.3-gke.2@sha256:abcd",
```

Comment:
please add test case to make sure we dont match more hostnames than expected. 

for example, these should not match:

 "anotherhost.gke.gcr.io/gcs-fuse-csi-driver-sidecar-mounter:v1.18.3-gke.2@sha256:abcd"

 "anotherhost/gke.gcr.io/gcs-fuse-csi-driver-sidecar-mounter:v1.18.3-gke.2@sha256:abcd"



Comment